### PR TITLE
fixes #56

### LINKED ISSUE #56: modules moeten kunnen lanceren zonder te verbinden met port 5000
de server module wil dat modules uitgevoerd kunnen worden, zelfs als ze niet kunnen connected met port 5000.

### PR DESCRIPTION
**Pull request by steviestickman**
currently python modules crashes when run unless manager/manager.py is running,
this pr fixes that flaw by crashing gracefullly, with error log and a hopefully helpfull error message.

### CODE REVIEW

**Comment by florianhumblot** [comment]
>fixes that flaw by crashing gracefullly 

It shoulnd't crash at all. It has to keep trying to reconnect. Otherwise issue #56 won't be fixed.

**Comment by steviestickman** [comment]
> Not fully working yet, Isha is working on the full fix.

the problem is when the module is already running, out of scope for this PR,
see #66 for details

**Comment by steviestickman** [comment]
florian said the fix works.
opening pr

File: client/comm.py [hunk 1]
```diff
@@ -93,7 +98,21 @@ class QueueManager(BaseManager):
 class Comm(BaseComm):
     def __init__(self):
         self.manager = QueueManager(address=BusConfig.ADDRESS, authkey=BusConfig.AUTH_KEY)
-        self.manager.connect()
+
```
Comment (sebastiaan1997) [inline client/comm.py]: Whats happens if the connection fails when the objecct is created?

Comment (steviestickman) [reply client/comm.py]: the connection is made on the call QueueManager.connect . not on object creation
we changed the connection to reattempt the connection once every 10 secconds.

**Pull request merged**